README.md

```diff
@@ -73,8 +73,8 @@ namespace Example
         // Retrieving your location IDs
         public void RetrieveLocations()
         {
-            LocationsApi _locationsApi = new LocationsApi();
-            var response = _locationsApi.ListLocations();
+            LocationApi _locationApi = new LocationApi();
```

Comment:
Ahh, a bug in the templates. Fixed in https://github.com/square/connect-api-specification/pull/57 and this repo addressed in https://github.com/square/connect-csharp-sdk/pull/31/commits/898e862dd023e000528fda40ea17c7b3962985d3